### PR TITLE
806: move plugin, resolution, magnetism and polydispersity docs to sasmodels

### PR DESCRIPTION
**Pull request by pkienzle**
As part of ticket #806, documentation about the the models should be in the sasmodels doc tree. 

This patch converts the equation images to latex markup in sphinx and copies the docs from the sasview fitting perspective into the sasmodels doc tree.  A corresponding patch is needed in sasview to remove the docs from the fitting perspective and instead pull them from sasmodels, but that can be done independently from this patch.

### CODE REVIEW

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

File: doc/guide/sesans/sans_to_sesans.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+First we calculate the Hankel transform including the absolute intensities by
+
+.. math:: G(\delta) = 2 \pi \int_0^{\infty} J_0(Q \delta) \frac{d \Sigma}{d \Omega} (Q) Q dQ \!,
+
+in which :math:`J_0` is the zeroth order Bessel function, :math:`\delta`
+the spin-echo length, :math:`Q` the wave vector transfer and :math:`\frac{d \Sigma}{d \Omega} (Q)`
+the scattering cross section in absolute units. This is a 1-dimensional
+integral, which can be rather fast. In the numerical calculation we integrate
+from :math:`Q_{min} = 0.1 \times 2 \pi / R_{max}` in which :math:`R_{max}`
+will be model dependent. We determined the factor 0.1 by varying its value
```
Comment (rprospero) [inline doc/guide/sesans/sans_to_sesans.rst]: I just added some changes to try and clear up the language.  This also brought to light that the sasview and sasmodels Sesans code are out of sync.  I'll get a pull request out for that immediately.

**Pull request merged**